@here/harp-mapview/lib/MapView.ts

```diff
@@ -2659,6 +2691,29 @@ export class MapView extends EventDispatcher {
             : this.m_raycaster.ray.intersectPlane(this.m_plane, cache.vector3[0]);
     }
 
+    /**
+     * Returns the {@link @here/harp-geoutils#GeoCoordinates} from the
+     * given screen position.
+     *
+     * @remarks
+     * This method will always return a point. if the given `(x, y)` value is not intersecting
+     * the ground plane a fallback point on the far plane will be computed.
+     * Therfore the returned point might not be on the earth surface (i.e. altitude > 0)
+     *
+     * @param x - The X position in css/client coordinates (without applied display ratio).
+     * @param y - The Y position in css/client coordinates (without applied display ratio).
+     * @param fallback - Fallback to
+     */
+    getGeoCoordinatesAtSafe(x: number, y: number): GeoCoordinates {
+        const worldPosition = this.getWorldPositionAtSafe(x, y);
+        const geoPos = this.projection.unprojectPoint(worldPosition);
+        if (!this.tileWrappingEnabled && this.projection.type === ProjectionType.Planar) {
+            // When the map is not wrapped we clamp the longitude
+            geoPos.longitude = THREE.MathUtils.clamp(geoPos.longitude, -180, 180);
```

Comment:
won't this be handled by the normalized function?

Reply:
no, normalize is for example converting a latitude of 181 to -179 what only makes sense if the world is repeated.

Reply:
Ok, I get it now, it should clamp, which isn't handled as you mentioned.